chroma_core/models/filesystem.py

```diff
@@ -97,6 +97,9 @@ def get_servers(self):
         # (http://stackoverflow.com/questions/4764110/django-template-cant-loop-defaultdict)
         return dict(servers)
 
+    def get_pools(self):
+        return OstPool.objects.filter(Q(filesystem=self))
```

Comment:
What sql does this create? Can (should) we be more specific?

Reply:
This should be returning all pools for the given filesystem.